src/frontend/app/store/actions/service-instances.actions.ts

```diff
@@ -0,0 +1,62 @@
+import { RequestOptions, URLSearchParams } from '@angular/http';
+import { schema } from 'normalizr';
+
+import { getAPIResourceGuid } from '../selectors/api.selectors';
+import { PaginationAction } from '../types/pagination.types';
+import { CFStartAction, ICFAction } from '../types/request.types';
+import { getActions } from './action.helper';
+import { ServiceInstancesSchema } from './action-types';
+
+export class GetServicesInstancesInSpace extends CFStartAction implements PaginationAction {
+  constructor(public cfGuid: string, public spaceGuid: string, public paginationKey: string) {
```

Comment:
`public cfGuid: string` --> `public endpointGuid: string`, otherwise will go out to all cf's. This will be the same for all CFStartActions